src/main/scala/karazin/scala/users/group/week2/homework/program.scala

```diff
@@ -1,33 +1,65 @@
 package karazin.scala.users.group.week2.homework
 
 // Do not forget to import custom implementation
+
 import adt._
 import model._
 import services._
 
 object program:
-  
+
   /*
    Print all view for all user's posts if they exists
   */
+
+  //implement with flatten to unbox ErrorOr and forEach
+  //to print every PostView with print method???????
+
   def printPostsViews(): ErrorOr[List[PostView]] = ???
- 
-  
+
   /*
    Getting view for all user's posts if they exists
   */
-  def getPostsViews(): ErrorOr[List[PostView]] = ???
+  def getPostsViews(): ErrorOr[List[ErrorOr[PostView]]] = {
```

Comment:
Compare the result type with the original one and do not cheat.

Reply:
I'm sorry, I didn't notice a bit.